api/all.yaml

```diff
@@ -1035,6 +1038,61 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ModelVersionInfo'
+  /permissions/{resource_type}/{name}:
+    parameters:
+      - name: resource_type
+        in: path
+        description: The type of the resource.
+        required: true
+        schema:
+          $ref: '#/components/schemas/ResourceType'
+      - name: name
+        in: path
+        description: The name of the principal.
```

Comment:
Shouldn't this be the name of the securable?

Reply:
Ok I've refactored the permissions yaml to more closely match databricks permissions api nomenclature across the board.